cmd/mcp-server/scenarios/container-oom-teardown.sh

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+require_tools oc
+
+APPS_NS=$(discover_apps_namespace)
```

Shellcheck (0.11.0)

[warning] APPS_NS appears unused. Verify use (or export if used externally).

(SC2034)